src/javascript/lib/landing_pages/ico.js

```diff
@@ -115,6 +115,13 @@ window.onload = function() {
         open_link.location = getLykkeReport(language.toLowerCase());
     });
 
+    document.getElementById('nishant-btn').addEventListener('click', function(e) {
+        e.preventDefault();
+        const open_link    = window.open();
+        open_link.opener   = null;
+        open_link.location = getNishantReport();
```

Comment:
if we are repeating this functionality everywhere can we add it as a function that expects a location?

Reply:
Done.